delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/Storages.scala

```diff
@@ -464,16 +435,6 @@ object Storages {
         } yield StorageUpdated(c.id, c.project, value, c.source, s.rev + 1, instant, c.subject)
     }
 
-    def tag(c: TagStorage) = state match {
```

Comment:
In `StorageState`, we should not have the tags anymore and the next method should only update the auditing fields
For the definition that lives in the same file, we can switch to `ScopedEntityDefinition.untagged`